src/mmw/apps/geoprocessing_api/calcs.py

```diff
@@ -87,8 +97,8 @@ def stream_data(results, geojson):
             "order": s['stream_order'],
             "lengthkm": s['lengthkm'],
             "ag_stream_pct": results['ag_stream_pct'],
-            # TODO: implement `slopepct`
-            "slopepct": 0,
+            "total_weighted_slope": s['slopesum'],
+            "avgslope": s['slopesum'] / s['lengthkm'] if s['slopesum'] else None  # noqa,
```

Comment:
Could we push the else onto a second line directly beneath the `if` rather than disabling the linter?

Reply:
It's silly but I just could not get the linter to be happy and have it look good. When the `if` goes to the next line the linter seems to want it lined up with the `"avgslope"`. Maybe we should make this a legit for-loop instead of in-lining it.

Reply:
What if we moved this out into a function and then called `calculateAvgSlope(s)` here?